doc/doxygen/doxyfile.in

```diff
@@ -8,6 +8,7 @@ PROJECT_BRIEF          =
 PROJECT_LOGO           =
 OUTPUT_DIRECTORY       = "@CMAKE_CURRENT_BINARY_DIR@"
 CREATE_SUBDIRS         = NO
+ALLOW_UNICODE_NAMES    = NO
```

Comment:
Is this the default order that doxygen generates? I asked because it looks pretty arbitrary and is not easy to find some config, without search functionality aid.

Reply:
It is, I ran `doxygen -u doxyfile.in` (like it says in the respective warning) and removed the comments to have the minimal changeset. You can test locally with the same command that the changes are these.